miri/parameters.py

```diff
@@ -194,12 +204,23 @@
                      'CV3'       : '2013-09-26T05:00:00',
                      'CV3_BURST' : '2015-08-01T00:00:00',
                      'CDP1'      : '2013-03-01T00:00:00',
+                     'SUB128'    : '2013-09-26T05:00:00',
```

Comment:
I'm not sure if this dictionary is used anywhere. From our description at https://wiki.miricle.org/Internal/Software/MiriCalfileMetaData, it looks like these values will be appropriate for any new CDP files produced based on CV3 data or newer (as well as valid for any non-burst subarrays from CV1 or CV2).

If anyone does still produce new files for burst subarrays from CV1 or CV2, then these values would not be appropriate.

And for "CDP9", I think we'll have a far more diverse range of USEAFTER values, as we'll be sweeping up all files produced at various points during commissioning. 

Still, I think it's an improvement to add these lines to the dictionary, but properly setting USEAFTER will still primarily be the responsibility of the CDP authors. Like you, I'm not sure anyone currently uses this dictionary.

Reply:
Actually, the dictionary is used. There is a convenience function set_referencefile_metadata() defined within the MIRI data models, which can be used to set reference file keywords (REFTYPE, AUTHOR, PEDIGREE, USEAFTER and DESCRIPTION). The useafter string can be set to a keyword or a date string. If 'TODAY' is provided, the USEAFTER keyword is set to today's date and time. If a string matching one of the keywords in this dictionary is provided it is translated into the equivalent date. And if the string contains a date it is written directly to the USEAFTER keyword. So any software using this convenience function would be using the dictionary. The "CDP9" entry is a placeholder which might need to be replaced by something else.

Reply:
Ah, that is a nice convenience function to have, I wasn't aware of it, thank you.